export class NotFoundError extends Error {
    constructor(){
        super('Recurso n√£o localizado')
        this.name = 'NotFoundError'
    }
}